component/mitz/README.md

```diff
@@ -0,0 +1,219 @@
+# MITZ Connector Component
+
+The MITZ Connector component enables integration with the Dutch national consent management system for healthcare. It handles consent subscription management and authorization queries.
+
+## Overview
+
+MITZ is a central consent management system in the Netherlands that allows patients to manage their healthcare data sharing preferences. This component provides:
+
+1. **Consent Subscription Management** - Subscribe to consent notifications for specific patient-provider-category combinations
+2. **Consent Authorization Queries** - Check if a healthcare provider has consent to access patient data
+3. **Notification Handling** - Receive and process consent change notifications from MITZ 
+
+## Architecture
+
+The component acts as a proxy between your application and the MITZ FHIR endpoints, handling:
+
+- mTLS authentication with client certificates
+- FHIR Subscription resource creation and management
+- XACML authorization decision queries
+- Configurable subscription notification endpoint
+
+## Configuration
+
+Configure the MITZ component in `knooppunt.yml`:
+
+```yaml
+mitz:
+  # MITZ base URL (paths are hardcoded in code)
+  mitzbase: "https://tst-api.mijn-mitz.nl"
+
+  # Endpoint URL where MITZ will send consent change notifications
+  notify_endpoint: "https://your-app.example.com/mitz/notify"
+
+  # Optional: Gateway and source system OIDs (added as extensions to subscriptions)
+  gateway_system: "urn:oid:2.16.840.1.113883.2.4.6.6.1"
+  source_system: "urn:oid:2.16.840.1.113883.2.4.6.6.90000017"
+
+  # mTLS client certificate configuration
+  tls_cert_file: "/path/to/client-cert.p12"
+  tls_key_password: "your-certificate-password"
+  tls_ca_file: "/path/to/ca-cert.pem"
+```
+
+### Configuration Options
+
+| Option | Required | Description |
+|--------|----------|-------------|
+| `mitzbase` | Yes | Base URL of the MITZ endpoint |
```

Comment:
Can you move these to `docs/CONFIGURATION.md`, under the "Configuration Options" chapter? That's the go-to place for users of the Knooppunt to find what to configure.

I'd say this markdown is aimed at developers (users won't find it)